docs/conceptual-bits.md

```diff
@@ -0,0 +1,69 @@
+# Conceptual bits that are useful for debugging Archivematica
+
+This page has some notes on conceptual bits that were useful as I was setting up/debugging our Archivematica deployment.
+Some of it is specific to Wellcome, some of it is general Archivematica knowledge.
+
+## Microservices/tasks/jobs
```

Comment:
Might make this heading:
```suggestion
## Microservices, tasks and jobs
```

As I thought you might be reffering to the same thing.